pages/linux/bwa.md

```diff
@@ -0,0 +1,24 @@
+# bwa
+
+> BWA is a short, low-divergent DNA sequences mapper against a large reference genome, such as the human genome.
+> More information: <https://github.com/lh3/bwa>.
+
+- Index reference genome:
+
+`bwa index {{path/to/reference.fa}}`
+
+- Map single-end reads (sequences) to indexed genome using 32 [t]*hreads and compress the result to save space:
+
+`bwa mem -t 32 {{path/to/reference.fa}} {{path/to/read_single_end.fq.gz}} | gzip -3 > {{path/to/alignment_single_end.sam.gz}}`
```

Comment:
We tend not to use commands which involve other commands, however given the complexity of the command, it might be a good idea to see what others think.

Reply:
Is it really necessary to specify the `-3` in the gzip command? If we left it out, the command would be more concise.

Reply:
I don't mind the change. this `-3` came from their examples posted in bwa repo.